docs/contributing/plugin_api/index.rst

```diff
@@ -1,6 +1,11 @@
 Plugin API
 ==========
 
+The Pulp Plugin API is versioned separately from Pulp Core. It is governed by `semantic
```

Comment:
I'm wondering where we declare this version? Is it in the setup.py maybe? How does that get declared in these docs? I'm thinking the API verison is `0.0.1a1dev0` or something like that.